app/data/ukis_main.json

```diff
@@ -0,0 +1,5112 @@
+{
+    "data_version": "0.0.2",
+    "description": "UK Innovation Survey",
+    "questionnaire_id": "",
+    "schema_version": "0.0.1",
+    "title": "UK Innovation Survey",
+    "navigation": true,
+    "survey_id": "uk-innovation-survey",
+    "groups": [
+        {
+            "blocks": [
+                {
+                    "type": "questionnaire",
+                    "id": "general-business-information-block-4",
+                    "sections": [
+                        {
+                            "description": "<p>3 year period 1 January 2014 to 31 December 2016.</p>",
+                            "number": "1",
+                            "questions": [
+                                {
+                                    "description": "",
+                                    "number": "1.1",
+                                    "title": "In which geographic markets did {{respondent.ru_name}} sell goods and/or services?",
+                                    "type": "General",
+                                    "id": "general-business-information-question-4",
+                                    "answers": [
+                                        {
+                                            "description": "",
+                                            "options": [
+                                                {
+                                                    "q_code": "0210",
+                                                    "value": "UK regional within approximately 100 miles of this business",
```

Comment:
Out of scope: It would be good to make value optional where the label is taken when value is not provided. I am not sure of a scenario where we have had a different value to label